googlemock/CMakeLists.txt

```diff
@@ -129,8 +136,15 @@ if(INSTALL_GMOCK)
     @ONLY)
   install(FILES "${CMAKE_BINARY_DIR}/gmock.pc" "${CMAKE_BINARY_DIR}/gmock_main.pc"
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+
+  # install CMake export files
+  install(EXPORT GtestTargets
+    FILE GmockConfig.cmake
+    NAMESPACE GTest::
+    DESTINATION cmake)
```

Comment:
`DESTINATION` should take `${CMAKE_INSTALL_LIBDIR}` into account. Otherwise its not consistent with the pkgconfig installation path and follows a path convention specfic to Windows systems (see [find_package](https://cmake.org/cmake/help/v3.8/command/find_package.html#command:find_package)). Therefore, change this line to:
`DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Gtest`
which is used on Windows and Unix systems.